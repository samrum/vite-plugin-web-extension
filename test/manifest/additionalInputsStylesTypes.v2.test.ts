import { getResourceDir, runManifestV2Test } from "./manifestTestUtils";

const resourceDir = getResourceDir("additionalInputsStylesTypes");

runManifestV2Test("additionalInputsStylesTypes", () => ({}), {
  additionalInputs: {
    styles: [`${resourceDir}/style1.css`, `${resourceDir}/style2.scss`],
  },
});
