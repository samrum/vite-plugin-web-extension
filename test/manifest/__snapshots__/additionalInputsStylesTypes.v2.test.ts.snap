// Vitest Snapshot v1

exports[`additionalInputsStylesTypes - Manifest V2 1`] = `
[
  {
    "fileName": "test/manifest/resources/additionalInputsStylesTypes/style1.css",
    "name": undefined,
    "source": "body {
  color: red;
}
",
    "type": "asset",
  },
  {
    "fileName": "test/manifest/resources/additionalInputsStylesTypes/style2.css",
    "name": undefined,
    "source": "body {
  color: green;
}",
    "type": "asset",
  },
  {
    "fileName": "manifest.json",
    "name": undefined,
    "source": "{
  \\"version\\": \\"1.0.0\\",
  \\"name\\": \\"Manifest Name\\",
  \\"manifest_version\\": 2,
  \\"web_accessible_resources\\": [
    \\"test/manifest/resources/additionalInputsStylesTypes/style1.css\\",
    \\"test/manifest/resources/additionalInputsStylesTypes/style2.css\\"
  ]
}",
    "type": "asset",
  },
]
`;
